libshaderc_spvc/include/shaderc/spvc.h

```diff
@@ -75,10 +75,10 @@ SHADERC_EXPORT void shaderc_spvc_compile_options_set_entry_point(
 SHADERC_EXPORT void shaderc_spvc_compile_options_set_remove_unused_variables(
     shaderc_spvc_compile_options_t options, bool b);
 
-// Sets the target shader environment, affecting which warnings or errors will
+// Sets the source shader environment, affecting which warnings or errors will
```

Comment:
Like in the C++ case, this should probably say the default is Vulkan 1.0.
(Pre-existing issue.)

Reply:
Done